.github/workflows/skms.yaml

```diff
@@ -0,0 +1,101 @@
+name: Create CMR in SKMS
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - skms-release-test
+
+jobs:
+  create-cmr:
+    # Only run this workflow on pull requests that have merged and not manually closed by user
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell for shell validation
+      - name: Run a one-line script
+        run: echo Starting CMR Action...
+
+      # Validate dependencies before continuing CMR creation
+      - name: Check Dependencies
+        run: |
+          if ! type "jq" >/dev/null; then
+            echo "jq is required but not installed"
+            exit 1
+          fi
+
+          if ! type "curl" >/dev/null; then
+            echo "curl is required but not installed"
+            exit 1
+          fi
+
+          echo "Dependencies check was successful"
+
+      # Run command for creating CMR in SKMS system
+      - run: |
+          start_date=$(TZ=":America/Los_Angeles" date '+%Y-%m-%d %H:%M')
+          DEFAULT_SKMS_URL='api.skms.adobe.com'
```

Comment:
For testing, it's better to use the stage env `stage.dev.skms.adobe.com`.

Reply:
I've switched to Production on purpose since there is an error that is blocking these machines with the production environment.  Thanks though for the reminder.